programs/stake/Cargo.toml

```diff
@@ -26,6 +27,7 @@ criterion = { workspace = true }
 proptest = { workspace = true }
 solana-compute-budget = { workspace = true }
 solana-logger = { workspace = true }
+solana-pubkey = { workspace = true }
```

Comment:
hmmm, is this redundant since `[dependencies]` already has it?

Reply:
yes, removed